src/components/common/ui/button-mui/Button.styles.ts

```diff
@@ -0,0 +1,197 @@
+import { SxProps, Theme } from '@mui/material/styles';
+export const button = (
+  color,
+  variant,
+  size,
+  startIcon,
+  endIcon,
+): SxProps<Theme> => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  borderRadius: '4px',
+  color: 'gray.600',
+  textTransform: 'none',
+  fontSize: '14px',
+  fontWeight: '700',
+  minWidth: 'unset',
+  minHeight: 'unset',
+  ...((startIcon || endIcon) && {
```

Comment:
move to all styles, cause it will only be applied when you have more than one child. and you can get rid of `startIcon` and `endIcon` in styles